packages/module-loader-commonjs/src/__tests__/index.test.ts

```diff
@@ -2,16 +2,23 @@
  * @jest-environment node
  */
 
+import {RequestInfo, RequestInit} from 'node-fetch';
+import ProxyAgent from 'proxy-agent';
 import {createCommonJsModuleLoader, loadCommonJsModule} from '..';
 
 let mockResponse: string;
+let requestUrl: RequestInfo;
+let requestInitOptions: RequestInit;
 
 // tslint:disable promise-function-async
-jest.mock('node-fetch', () => () =>
-  Promise.resolve({
+jest.mock('node-fetch', () => (url: RequestInfo, options: RequestInit) => {
```

Comment:
Please change parameter names to `(url: RequestInfo, init: RequestInit) =>`

Reply:
@clebert ja, better, done!